glide.lock

```diff
@@ -13,7 +13,7 @@ imports:
 - name: github.com/andreyvit/diff
   version: c7f18ee00883bfd3b00e0a2bf7607827e0148ad4
 - name: github.com/apparentlymart/go-cidr
-  version: c0fb5fbe0acb592411e2db59add389a43260ad44
+  version: d07067359aee8ee9101c1a79a4f286cf0cea059f
```

Comment:
is there an easy way to find out why this dep was updated?

Reply:
I thought glide needs manual dependency updats. I would remove those manually. Does library-go have the same changes?

Reply:
I decided to pin them manually. Library-go does have some of them but no all. 
Anyway, now the diff is much smaller.